superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -73,7 +73,7 @@ superset_docker_restart_policy: "Always"
 superset_service_port: "32443"
 superset_nginx_port: "32443"
 
-SUPERSET_NPROCESSORS: 1
+SUPERSET_NPROCESSORS: 4
```

Comment:
We should relay on extra_vars for modifing these defaults. As on low footprint machine we would not like to increase the same

Reply:
Reverted.